homeassistant/components/homewizard/sensor.py

```diff
@@ -41,6 +49,11 @@
         icon="mdi:wifi",
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
+    SensorEntityDescription(
+        key="active_tariff",
+        name="Active tariff",
+        icon="mdi:calendar-clock",
+    ),
```

Comment:
Currently it is just `1` or `2`, because NL and BE tariffs are swapped, we can't easily convert this to 'Dal' and 'Normaal' tariff because of that. Should we keep it raw or do we want to make it nicer for the user somehow?